test/test_nested_tensor_masking.py

```diff
@@ -3,42 +3,257 @@
 import pdb
 import sys
 import torch
-import nestedtensor
+import nestedtensor as nt
 import unittest
 from utils import TestCase
 import random
-
 import utils
+from utils import nested_size_to_list
+
+#
+# Test data
+#
+original_nt = nt.nested_tensor([
```

Comment:
I think there's two arguments to be made for inlining them

a) readability
b) allowing to explicitly test in-place vs. out-of-place changes. Can a mask every be a view? Should it be? etc.

Reply:
Moved them into the tests